packages/react-core/src/next/components/Wizard/examples/WizardEnabledOnFormValidation.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { Form, FormGroup, TextInput } from '@patternfly/react-core';
+import { Wizard, WizardStep } from '@patternfly/react-core/next';
+
+interface SampleFormProps {
+  value: string;
+  isValid: boolean;
+  setValue: (value: string) => void;
+  setIsValid: (isValid: boolean) => void;
+}
+
+const SampleForm: React.FunctionComponent<SampleFormProps> = ({ value, isValid, setValue, setIsValid }) => {
+  const validated = isValid ? 'default' : 'error';
+
+  const handleTextInputChange = (value: string) => {
+    const isValid = /^\d+$/.test(value);
+
+    setValue(value);
+    setIsValid(isValid);
+  };
+
+  return (
+    <Form>
+      <FormGroup
+        label="Age:"
+        type="number"
+        helperTextInvalid="Age has to be a number"
```

Comment:
FWIW the existing demo has `helperText` and this one doesn't - was that intentional? 

https://github.com/patternfly/patternfly-react/blob/90e75e7a9051bbb604234a705b3ecbcecbdf22ea/packages/react-core/src/components/Wizard/examples/WizardEnabledOnFormValidation.tsx#L32-L33